pixeltable/share/publish.py

```diff
@@ -66,6 +65,18 @@ def publish_snapshot(dest_tbl_uri: str, src_tbl: pxt.Table) -> str:
     return confirmed_tbl_uri
 
 
+def clone_snapshot(dest_tbl_uri: str) -> list[FullTableMd]:
+    headers_json = {'X-api-key': Env.get().pxt_api_key, 'Content-Type': 'application/json'}
+    clone_request_json = {'operation_type': 'clone_snapshot', 'table_uri': dest_tbl_uri}
+    response = requests.post(PIXELTABLE_API_URL, json=clone_request_json, headers=headers_json)
+    if response.status_code != 200:
+        raise excs.Error(f'Error cloning snapshot: {response.text}')
+    response_json = response.json()
+    if not isinstance(response_json, dict) or 'table_uri' not in response_json:
+        raise excs.Error(f'Error unexpected response from server.\n{response_json}')
+    return [FullTableMd.from_dict(t) for t in response_json['md']['tables']]
```

Comment:
Where is the S3 location? Is it now inside the `additional_md` somewhere?

[Note to @mkornacker : `clone_snapshot()` will also need to call `create_replica()` and load data into the tables. That will be done as part of my upcoming catalog ops PR and I'll incorporate it here on top of Amit's changes.]

Reply:
its in additional_md of table_md of the replica (first table in tables)